### PR TITLE
Refine PR and Issue templates

### PR DESCRIPTION
**Pull request by inancgumus**
## What?

1. Refines the PR template.
2. Add an issue template for the internal team.
3. Updates the community forum link.
4. Adds the migration guide link to the new issue page.

I can't preview these templates, so IDK how they will look when rendered. **Update:** I found a suboptimal way. Go to the issue template file on the Github UI.

## Why?

To provide better workflow both for internal and external contributors.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml
Comment (inancgumus) [inline .github/ISSUE_TEMPLATE/config.yml:11]: I'm not sure about saying `k6 v0.46+` as we're on the xk6-browser repo. Any suggestions?

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (inancgumus) [inline .github/ISSUE_TEMPLATE/team_issues.yaml:2]: This issue is only for the xk6-browser team. It won't show up on the new issue screen. IDK how we can use it, though :) I suggest checking it out once we merge this PR and amending it later on if it doesn't work.

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (inancgumus) [inline .github/ISSUE_TEMPLATE/team_issues.yaml:20]: I put this here to not to convolute the commented sections below.

File: .github/ISSUE_TEMPLATE/config.yml
Comment (inancgumus) [inline .github/ISSUE_TEMPLATE/config.yml:3]: Since we're already on the xk6-browser repo, I believe we should need to repeat the repo name here.

File: .github/ISSUE_TEMPLATE/config.yml
Comment (ka3de) [reply .github/ISSUE_TEMPLATE/config.yml:11]: I think this is OK. I would not expect too many users being aware of xk6-browser version and using it as an extension now that it's bundled with k6. So I think referencing k6 version directly is probably the best :+1:

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (ka3de) [inline .github/ISSUE_TEMPLATE/team_issues.yaml:20]: Can you clarify what is the intention of this `Suggestion` section?

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (ka3de) [inline .github/ISSUE_TEMPLATE/team_issues.yaml:30]: Should this suggest a possible list of subissues? E.g.:
```
- [ ] Subissue 1
- [ ] Subissue 2
- [ ] Subissue i
```
Or something similar? This does not seem the best option either :thinking:

**Review by ka3de — COMMENTED** [review]
Thanks for taking care of this @inancgumus ! :raised_hands: 
Just a few considerations.

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (inancgumus) [reply .github/ISSUE_TEMPLATE/team_issues.yaml:30]: I got your intention, and it’s a good idea. Since this is an internal issue only for the team, IDK if it matters much. A single item can be enough for us to remember to put issues there. WDYT?

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (inancgumus) [reply .github/ISSUE_TEMPLATE/team_issues.yaml:20]: Something like this https://github.com/grafana/xk6-browser/issues/1025. See the suggestion section there.

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (ka3de) [reply .github/ISSUE_TEMPLATE/team_issues.yaml:30]: Right, if we can keep this internal it is OK for me :+1:

**Review by ka3de — APPROVED** [review]
LGTM.

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (ankur22) [reply .github/ISSUE_TEMPLATE/team_issues.yaml:20]: Since you have "What" and "Why", you could change this to "How" instead of "Suggestion".

The description could be reworded to something like "Do you have any suggestions/solutions on how to solve the issue?"

**Review by ankur22 — APPROVED** [review]
Thanks for tackling this. Hopefully we will have a consistent way of creating issues now 🙂 LGTM 👍

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (inancgumus) [reply .github/ISSUE_TEMPLATE/team_issues.yaml:20]: I liked this one, thx!

File: .github/ISSUE_TEMPLATE/team_issues.yaml
Comment (ankur22) [inline .github/ISSUE_TEMPLATE/team_issues.yaml:19]:
```suggestion
      label: How?
```

Comment (inancgumus) [reply .github/ISSUE_TEMPLATE/team_issues.yaml:19]: Oops 🤦

Comment (inancgumus) [reply .github/ISSUE_TEMPLATE/team_issues.yaml]: Fixed in https://github.com/grafana/xk6-browser/pull/1024/commits/bca88d498c27e63022e0fa243f5230673a802ff5.

**Review by ankur22 — APPROVED** [review]
LGTM 👍

**Pull request merged**
